### PR TITLE
Make the banner component

### PR DESCRIPTION
**Pull request by tim-s-ccs**
- Make the banner component
- Add example and README for the component

This is part of the ticket [OST-134](https://crowncommercialservice.atlassian.net/browse/OST-134) (2/5)

Once this has been merged we will need to create a new release (3.1.0) and then update all the frontend apps that use this package.

### CODE REVIEW

File: src/digitalmarketplace/components/new-framework-banner/new-framework-banner.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+params:
+examples:
+  - name: default
+    description: A banner to inform users where to sign up for the new DOS6 framework.
```
Comment (bjgill) [inline src/digitalmarketplace/components/new-framework-banner/new-framework-banner.yaml:4]: Why do we need a description for the example?

**Review by bjgill — COMMENTED** [review]
Looks good. Just one minor question. ~Also, do we need to bump the version?~

File: src/digitalmarketplace/components/new-framework-banner/template.njk [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{%- from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner -%}
+
+{% set html %}
+  <p class="govuk-notification-banner__heading">
+    Supplier information for Digital Outcomes 6 is not available from this log in screen.
```
Comment (bjgill) [inline src/digitalmarketplace/components/new-framework-banner/template.njk:5]: > from this log in screen

Ah - just had a thought. Where are we showing this banner? Is this text valid for all places?

Comment (tim-s-ccs) [reply src/digitalmarketplace/components/new-framework-banner/template.njk]: The text will now be:
_Supplier information for Digital Outcomes 6 is not available from this page. Start or continue your application for the Digital Outcomes 6 framework._

**Review by bjgill — APPROVED** [review]
:shipit:

**Pull request merged**